deploy/dev.yml

```diff
@@ -0,0 +1,107 @@
+version: "3"
+services:
+    dotcom:
+        build:
+            context: ../
+            dockerfile: ./deploy/dotcom/dev/Dockerfile
+        env_file:
+            - ../.env
+        depends_on:
+            - redis-cluster-init
+        environment:
+            - MIX_ENV=dev
+        expose:
+            - 4001
+        ports:
+            - 4001:4001
+            - 8090:8090
+        volumes:
+            - ../:/app
+        networks:
+            redis_network:
+                ipv4_address: 10.0.0.2
+    monitor:
+        build:
+            context: ../
+            dockerfile: ./deploy/monitor/Dockerfile
+        env_file:
+            - ../.env
```

Comment:
I don't think the monitor uses any of the environment variables?

Reply:
Yeah, all of the healthchecks do. E.g., https://github.com/mbta/dotcom/blob/fcca6a8aa967b21b6f7a457c8ab1b82c90ed0ad8/integration/health_checks/cms.js#L4